crates/storage/src/store/multistore.rs

```diff
@@ -0,0 +1,69 @@
+use std::sync::Arc;
+
+use super::substore::SubstoreConfig;
+
+/// A collection of substore, each with a unique prefix.
+#[derive(Debug, Clone)]
+pub struct Multistore {
+    transparent_store: Arc<SubstoreConfig>,
```

Comment:
I'm not sure `transparent_store` is the best name here. Maybe `root_store` would be better, since it's at the root of the tree?